RecoLocalMuon/GEMSegment/python/me0Segments_cfi.py

```diff
@@ -11,8 +11,8 @@
         dYclusBoxMax = cms.double(5.),
         preClusteringUseChaining = cms.bool(True),
         dPhiChainBoxMax = cms.double(.02),
-        dEtaChainBoxMax = cms.double(.05),
-        dTimeChainBoxMax = cms.double(1.50), # 1ns, +/- time to fly through 30cm thick ME0
+        dEtaChainBoxMax = cms.double(.15),
+        dTimeChainBoxMax = cms.double(32.0), # 1ns, +/- time to fly through 30cm thick ME0
```

Comment:
Is this an equivalent of +/- 1 BX cut for discrete values (-25, 0, 25, ...)?
In this case better set it to 25.05 or something like that with an appropriate comment.

If ME0 hit tof is known more precisely, then 32 looks wrong.
At least the comment should be modified.